ResidentInformationApi/V1/Gateways/AcademyInformationGateWay.cs

```diff
@@ -0,0 +1,31 @@
+using System;
```

Comment:
the file name should be `AcademyInformationGateway`

Reply:
Amended!